frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -54,17 +54,19 @@ const menuActions = [
   ModifyLabels,
   ModifyAnnotations,
   Edit,
-  (kind, obj) => ({
+  (kind, pvc) => ({
     label: i18next.t('public~Delete PersistentVolumeClaim'),
     callback: () =>
       deletePVCModal({
-        pvc: obj,
+        kind,
+        pvc,
+        redirectTo: `/k8s/ns/${pvc.metadata.namespace}/${PersistentVolumeClaimModel.plural}`,
```

Comment:
Why is `redirectTo` needed? It looks like we changed the default behavior to go to the list page now.

Reply:
Avoid building paths manually. Use one of the helpers instead.

Reply:
> Why is `redirectTo` needed? It looks like we changed the default behavior to go to the list page now.

@spadgett I tested in 4.9, it is not redirecting after delete action.

Reply:
> Avoid building paths manually. Use one of the helpers instead.

Noted. Thanks

Reply:
> > Why is `redirectTo` needed? It looks like we changed the default behavior to go to the list page now.
> 
> @spadgett I tested in 4.9, it is not redirecting after delete action.

The PVC is not using the common delete modal that has redirect.